### PR TITLE
Update dotnet-trace.md with clreventlevel values

### LINKED ISSUE #36129: dotnet-trace:  possible `--clreventlevel` values not documented
Possible values for the `--clreventlevel` option are not documented here.  I had to dig around to find https://learn.microsoft.com/en-us/dotnet/framework/performance/clr-etw-keywords-and-levels#etw-event-levels.  It would be helpful if this documentation directly listed possible values for the option or linked to that documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 57216dad-d87d-eeef-adc8-82bfe4251343
* Version Independent ID: 3e7ab61f-5a06-b92c-5c74-0fae43adb35e
* Content: [dotnet-trace diagnostic tool - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace#dotnet-trace-collect)
* Content Source: [docs/core/diagnostics/dotnet-trace.md](https://github.com/dotnet/docs/blob/main/docs/core/diagnostics/dotnet-trace.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tommcdon
* Microsoft Alias: **tommcdon**

### PR DESCRIPTION
**Pull request by Johanneshn**
## Summary

Adds a table to dotnet-trace-md with possible values for the `--clreventlevel <clreventlevel>` option.

Fixes #36129

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/diagnostics/dotnet-trace.md](https://github.com/dotnet/docs/blob/0c2267a0a84d1c9156c495423c6efbc908f68fdd/docs/core/diagnostics/dotnet-trace.md) | [dotnet-trace performance analysis utility](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace?branch=pr-en-us-36375) |

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

File: docs/core/diagnostics/dotnet-trace.md [hunk 1]
```diff
@@ -98,6 +98,16 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
...
+  The table belows show the list of avaiable event levels:
+  
+  | String Value | Numberic Value |
+  | ------------ | ------------------- |
+  | `logalways` | `0` |
+  | `critical` | `1` |
+  | `error` | `2` |
+  | `warning` | `3` |
+  | `informational` | `4` |
+  | `verbose` | `5` |
```
Comment (BartoszKlonowski) [inline docs/core/diagnostics/dotnet-trace.md:110]: Just a nit:
Usually the good practice I observe in this code base (and I believe: in general), is to justify the raw text (the table implementation) the way it should look like when actually displaying. Such formatting ease the further development, and improves the readability.
So, I would suggest something like below:
```suggestion
  | String Value    | Numeric Value |
  | --------------- | :-----------: |
  | `logalways`     |      `0`      |
  | `critical`      |      `1`      |
  | `error`         |      `2`      |
  | `warning`       |      `3`      |
  | `informational` |      `4`      |
  | `verbose`       |      `5`      |
```
Note I also justify the `Numeric value` column as center, and fixed the `Numberic` typo.

**Review by BartoszKlonowski — COMMENTED** [review]
There is a typo, I believe, in "Num<b>(b)</b>eric Value" column title.
I also left a nitpicking suggestion for improved layout.

File: docs/core/diagnostics/dotnet-trace.md [hunk 2]
```diff
@@ -98,6 +98,16 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
 - **`--clreventlevel <clreventlevel>`**
 
   Verbosity of CLR events to be emitted.
+  The table belows show the list of avaiable event levels:
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:101]:
```suggestion
  The following table show the available event levels.
```

File: docs/core/diagnostics/dotnet-trace.md [hunk 3]
```diff
@@ -98,6 +98,16 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
...
+  
+  | String Value | Numberic Value |
```
Comment (gewarren) [inline docs/core/diagnostics/dotnet-trace.md:103]:
```suggestion
  | String value | Numeric value |
```

File: docs/core/diagnostics/dotnet-trace.md (hunk 2 above)
Comment (BartoszKlonowski) [reply docs/core/diagnostics/dotnet-trace.md:101]: Shouldn't there be "table shows"?

Comment (gewarren) [reply docs/core/diagnostics/dotnet-trace.md:101]: Fixed my comment, thanks.

**Review by gewarren — APPROVED** [review]

**Pull request merged**